packages/web-components/src/components/masthead/masthead-composite.ts

```diff
@@ -140,10 +141,10 @@ class DDSMastheadComposite extends HostListenerMixin(LitElement) {
    * @param sections menu section data object
    */
   // eslint-disable-next-line class-methods-use-this
-  protected _getHighlightedMenuItems(sections) {
+  protected _getMenuItems(sections) {
```

Comment:
I think renaming this function may have broken the "selected menu item" highlighting.
<img width="850" alt="Screenshot 2022-12-05 at 1 52 05 PM" src="https://user-images.githubusercontent.com/25532785/205719186-85ba45a4-38e4-4ec8-8c6f-f3bb65dde3ca.png">